### PR TITLE
fix agent.config

### PR DESCRIPTION
**Pull request by dashanji**
### Fix <bug description or the bug issue number or bug issue link>
- [ ] Add a unit test to verify that the fix works.
- [x] Explain briefly why the bug exists and how to fix it.

Follow the doc to readjust the order of agent.config and fix the mismatched naming between the doc and the agent.config.

### CODE REVIEW

File: apm-sniffer/config/agent.config [hunk 1]
```diff
@@ -219,7 +222,7 @@ plugin.dubbo.consumer_arguments_length_threshold=${SW_PLUGIN_DUBBO_CONSUMER_ARGU
 #  Apache Dubbo provider collect `arguments` in RPC call, use `Object#toString` to collect `arguments`. 
 plugin.dubbo.collect_provider_arguments=${SW_PLUGIN_DUBBO_COLLECT_PROVIDER_ARGUMENTS:false}
 #  When `plugin.dubbo.provider_consumer_arguments` is `true`, Arguments of length from the front will to the OAP backend 
-plugin.dubbo.consumer_provider_length_threshold=${SW_PLUGIN_DUBBO_CONSUMER_PROVIDER_LENGTH_THRESHOLD:256}
```
Comment (kezhenxu94) [inline apm-sniffer/config/agent.config:222]: Also update the typo in line 221

Comment (dashanji) [reply apm-sniffer/config/agent.config:222]: Ok.

**Comment by wu-sheng** [comment]
FYI @kezhenxu94 When you have time, let's add some skip mechanism back to this new repo.

File: apm-sniffer/config/agent.config [hunk 2]
```diff
@@ -20,17 +20,16 @@ agent.namespace=${SW_AGENT_NAMESPACE:}
 # The service name in UI
 agent.service_name=${SW_AGENT_NAME:Your_ApplicationName}
 
-# Instance name is the identity of an instance, should be unique in the service. If empty, SkyWalking agent will
-# generate an 32-bit uuid. BY Default, SkyWalking uses UUID@hostname as the instance name. Max length is 50(UTF-8 char)
-agent.instance_name=${SW_AGENT_INSTANCE_NAME:}
-
```
Comment (wu-sheng) [inline apm-sniffer/config/agent.config:26]: Why is this removed?

**Review by wu-sheng — APPROVED** [review]
One more thing, it seems all system environment variables are not list in `configurations.md`. We should update them. @dashanji Are you willing to do that in another PR?

File: docs/menu.yml [hunk 3]
```diff
@@ -28,14 +28,14 @@ catalog:
...
-      - name: "Dynamic Configurations"
-        path: "/en/setup/service-agent/java-agent/configurations#dynamic-configurations"
       - name: "Optional Plugins"
         path: "/en/setup/service-agent/java-agent/readme#optional-plugins"
       - name: "Bootstrap/JVM class plugin"
         path: "/en/setup/service-agent/java-agent/readme#bootstrap-class-plugins"
       - name: "Agent Configuration Properties"
         path: "/en/setup/service-agent/java-agent/configurations#table-of-agent-configuration-properties"
+      - name: "Dynamic Configurations"
+        path: "/en/setup/service-agent/java-agent/configurations#dynamic-configurations"
```
Comment (wu-sheng) [inline docs/menu.yml:38]: I did this.

**Review by wu-sheng — APPROVED** [review]

**Review by kezhenxu94 — APPROVED** [review]
